app/static/messages.properties

```diff
@@ -1,6 +1,7 @@
 {
     "aws_label" : "AWS",
     "azure_label" : "Azure",
+    "azure_rm_label" : "Azure RM",
```

Comment:
Please use only 'Azure' in manage credentials, in order to allow to create ARM related credentials